### ISSUE TITLE
[v-select] Incorrect error: 'item-color' has been replaced with 'item-props.color' 

### ISSUE THREAD

**Issue by wg-kim** [vuetifyjs/eslint-plugin-vuetify#82]
# report 
When I use the v-select's item-color prop, there is an  an Incorrect error.

```
 'item-color' has been replaced with 'item-props.color' 
```
![image](https://github.com/vuetifyjs/eslint-plugin-vuetify/assets/16056052/e2c8aece-40e0-4983-b8f7-baea8ea81416)

# referer
The 'item-color' is now on use.
https://vuetifyjs.com/en/api/v-select/#links

**Comment by wg-kim** [comment]
@KaelWD 
Thanks!
